src/aiortc/contrib/media.py

```diff
@@ -18,7 +18,8 @@
 from ..mediastreams import AUDIO_PTIME, MediaStreamError, MediaStreamTrack, KeypointsFrame
 
 from first_order_model.fom_wrapper import FirstOrderModel
-from first_order_model.reconstruction import frame_to_tensor
+from first_order_model.reconstruction import frame_to_tensor, resize_tensor_to_array
```

Comment:
can you point me to the appropriate branch in nets_implementation so I can actually validate these changes? Also, this is going to break if we expect a new checkpoint of the submodule for this to work.

Reply:
Here is the function
https://github.mit.edu/NeTS/nets_implementation/blob/af490909ec085e91ee938928419b023a4155163f/first_order_model/utils.py#L24